src/build.gradle

```diff
@@ -97,7 +97,12 @@ subprojects {
         compile "org.slf4j:slf4j-simple:1.7.25"
     }
 
-    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    compileKotlin {
+        kotlinOptions {
+            jvmTarget = "1.8"
+        }
+    }
+    compileTestKotlin {
         kotlinOptions {
```

Comment:
This change isn't strictly necessary - it's just a refactor to improve readability.